xpk.py

```diff
@@ -2008,6 +2059,14 @@ def workload_create(args) -> int:
         f' To view the metric data for your workload, select {args.workload} from the JobName filter on the dashboard.'
     )
 
+  if debugging_dashboard_id is not None:
```

Comment:
Should this be behind the `--deploy-stacktrace-sidecar` flag as well?

Reply:
It won't be needed, because if the flag is not set then `debugging_dashboard_id` will be `None`. In that case, we won't see any logs.

Reply:
I might be misunderstanding - but isn't debugging_dashboard_id tied to whether the user has separately run the terraform script? If so, wouldn't it be possible for users to have that setup in their project, but to not have stack traces available because they weren't forwarded with the `--deploy-stacktrace-sidecar` flag?

Reply:
For GKE workloads, debugging dashboard won't display any stack traces for the workload unless user opt to forward the logs using `deploy-stacktrace-sidecar`. So, it doesn't make any sense to output the link to debugging dashboard when this flag is not enabled because there won't be anything to check.

Reply:
Yes, 100% agreed on not showing the link if the flag isn't enabled. I missed seeing where debugging_dashboard_id was set to None in the code. I see it now before making the call to get_gke_debugging_dashboard. Thanks Surbhi